02/styles/global.css

```diff
@@ -1,26 +1,52 @@
 * {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+body {
+	display: grid;
+	height: 100vh;
 }
 
-.a, .b, .c {
-    border: 3px solid transparent;
-    min-height: 50px;
-
-    display: flex;
-    justify-content: center;
-    align-items: center;
+.a,
+.b,
+.c {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	max-width: 1200px;
+	border: 5px solid transparent;
+	width: 100%;
+	min-height: 50px;
+	margin: 0 auto;
 }
 
 .a {
-    border-color: #ffffba
+	height: 50%;
+	border-color: #ffffba;
 }
 
 .b {
-    border-color: #ffdfba;
+	height: 50%;
+	border-color: #ffdfba;
 }
 
 .c {
-    border-color: #baffc9;
-}
\ No newline at end of file
+	grid-row: 3 / 3;
+	height: 50%;
+	border-color: #baffc9;
+}
+@media (max-width: 601px) {
```

Comment:
Raczej odwrotnie - jak przekroczy czyli jak ma `min-width: 601` to wtedy mają być 2 kolumny :)